packages/rancher-vsphere-csi/charts/questions.yaml

```diff
@@ -110,6 +110,11 @@ questions:
         default: true
         type: boolean
 
+      - variable: storageClass.allowVolumeExpansion
+        label: Allow Volume Expansion
+        description: Allows resizing the volume by editing the corresponding PVC object
+        type: string
```

Comment:
Why is the type `string`, not `boolean`?

Reply:
Good catch, fixed